src/esp/physics/bullet/BulletPhysicsManager.h

```diff
@@ -153,6 +153,8 @@ class BulletPhysicsManager : public PhysicsManager {
    */
   double getSceneRestitutionCoefficient();
 
+  virtual void debugDraw(Magnum::Matrix4 projTrans);
```

Comment:
const?

All of the getters declared above here should be const.

Reply:
Agreed that getters should be const, created a PR for this #303. However, this function modifies the internal state of the debugger object, so it shouldn't be const. I did correct the input to const ref, though.

Reply:
I guess it depends on whether you consider the debugger object part of the observable state of the physics manager. If it wasn't part of the observable state, you could make it mutable. Not sure what the case is here.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#con2-by-default-make-member-functions-const
https://isocpp.org/wiki/faq/const-correctness#mutable-data-members

Reply:
Good point. Made a PR for this #306. 👍